src/DssSpell.sol

```diff
@@ -160,40 +86,26 @@ contract DssSpellAction is DssAction {
         );
     }
 
-    function _onboardRWA015A() internal {
-        bytes32 ilk = "RWA015-A";
-
-        // Init the RwaLiquidationOracle
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(ilk, RWA015_A_INITIAL_PRICE, RWA015_A_DOC, RWA015_A_TAU);
-        (, address pip, , ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
-
-        // Init RWA015 in Vat
-        Initializable(MCD_VAT).init(ilk);
-        // Init RWA015 in Jug
-        Initializable(MCD_JUG).init(ilk);
-
-        // Allow RWA015 Join to modify Vat registry
-        DssExecLib.authorize(MCD_VAT, MCD_JOIN_RWA015_A);
-
-        // Stability Fee is 0 for this ilk
-
-        // 2_500_000 debt ceiling
-        DssExecLib.increaseIlkDebtCeiling(ilk, RWA015_A_LINE, /* _global = */ true);
+    
+    function actions() public override {
 
-        // Set price feed for RWA015
-        DssExecLib.setContract(MCD_SPOT, ilk, "pip", pip);
+        // --- Activate Andromeda Autoline ---
+        // Forum: https://forum.makerdao.com/t/consolidated-action-items-for-2023-06-28-executive/21187
+        // Forum: https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974
 
-        // Set minimum collateralization ratio
-        DssExecLib.setIlkLiquidationRatio(ilk, RWA015_A_MAT);
+        // Activate autoline with line 1.28 billion DAI, gap 50 million DAI, ttl 86400
+        DssExecLib.setIlkAutoLineParameters("RWA015-A", 1_280 * MILLION, 50 * MILLION, 24 hours);
 
-        // Poke the spotter to pull in a price
-        DssExecLib.updateCollateralPrice(ilk);
+        // Bump Oracle Price to 1.28 billion DAI
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
+                "RWA015-A",
+                 1_280 * MILLION * WAD
+        );
```

Comment:
do you think it would be appropriate to always specify in the comment how the value for the bump has been derived? e.g. [here](https://github.com/makerdao/spells-goerli/blob/bd3c2de0b0527a79691aa2b2ba22a04536a42c28/src/DssSpell.sol#L297) its different

Reply:
For this ilk we have 0 SF, so price will be exactly the same as DC.

But if you think it make sense i can add a formula here

Reply:
> For this ilk we have 0 SF, so price will be exactly the same as DC.

yes , this is clear and makes sense. 

> But if you think it make sense i can add a formula here

i would go with the approach that is more explicit to the reader ( = reason is listed right in the code ). So please add the comment about 0 SF for clarity.